src/core/declarativewebcontainer.cpp

```diff
@@ -743,11 +746,14 @@ void DeclarativeWebContainer::initialize()
         return;
     }
 
+    // From this point onwards, we're ready to initialize.
```

Comment:
Maybe add that as a second-sentence to the comment, something like: "We set m_initialized to true prior to the block below since we may need to call loadTab() within it, and that function is guarded by the value of m_initialized."